File: src/main/java/xyz/chunkstories/api/server/PermissionsManager.java
//
// This file is a part of the Chunk Stories API codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package xyz.chunkstories.api.server;

import xyz.chunkstories.api.player.Player;

public interface PermissionsManager {
	public boolean hasPermission(Player player, String permissionNode);
}

File: src/main/java/xyz/chunkstories/api/server/PermissionsManager.kt
//
// This file is a part of the Chunk Stories API codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package xyz.chunkstories.api.server

import xyz.chunkstories.api.player.Player

interface PermissionsManager {
    fun hasPermission(player: Player, permissionNode: String): Boolean
}
